service/src/test/resources/ehr/mapper/immunization/expected-output-observation-statement-no-information.xml

```diff
@@ -0,0 +1,16 @@
+<component typeCode="COMP" >
+    <ObservationStatment classCode="OBS" moodCode="EVN">
+        <id root="C93659E1-1107-441C-BE25-C5EF4B7831D1" />
+        <statusCode code="COMPLETE" />
+        <effectiveTime>
+            <center value="20100630055900"/>
+        </effectiveTime>
+        <availabilityTime value="20100630054806" />
+        <pertinentInformation typeCode="PERT">
+            <sequenceNumber value="+1"/>
+            <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                <text></text>
```

Comment:
Not sure if this will ever be empty. Property primarySource has Card. 1..1 so it should be included. Confirm if this is the case. If it is possible to have empty text, confirm if we output as it is here or if pertinentInformation should be suppressed entirely.

Reply:
Pertinent information object no longer created if no string associated